sentry-android-core/src/main/java/io/sentry/android/core/performance/AppStartMetrics.java

```diff
@@ -153,17 +155,7 @@ public void onAppStartSpansSent() {
   }
 
   public boolean shouldSendStartMeasurements() {
-    return shouldSendStartMeasurements;
-  }
-
-  public void restartAppStart(final long uptimeMillis) {
```

Comment:
Restore this method and call it when we check the duration and foreground flag in the activity creation.
This way we can reset all flags and timestamps correctly
See comment below

Reply:
👍 I've inlined it